app/templates/eventStudentView.html

```diff
@@ -0,0 +1,97 @@
+{% set title = "View Event" %}
```

Comment:
Let's make this the name of the event instead of something general

Reply:
resolved